recipe_scrapers/justbento.py

```diff
@@ -7,46 +9,47 @@ class JustBento(AbstractScraper):
     def host(cls):
         return "justbento.com"
 
-    def title(self):
+    def title(self) -> Optional[str]:
         expected_prefix = "Recipe: "
         title = self.soup.find("meta", {"property": "og:title", "content": True})
         return title.get("content").replace(expected_prefix, "")
 
-    def total_time(self):
+    def total_time(self) -> Optional[int]:
         time = self.soup.find(
             "div", {"class": "field-name-taxonomy-vocabulary-2"}
         ).find("a", {"typeof": "skos:Concept"})
         return get_minutes(time)
 
-    def yields(self):
+    def yields(self) -> Optional[str]:
         return "1"
 
-    def ingredients(self):
+    def ingredients(self) -> Optional[List[str]]:
         ingredients = (
             self.soup.find("div", {"class": "field-name-body"}).find("ul").findAll("li")
         )
         return [normalize_string(ingredient.get_text()) for ingredient in ingredients]
 
-    def instructions(self):
+    def instructions(self) -> Optional[str]:
         elements_after_title = (
             self.soup.find("div", {"class": "field-name-body"})
             .find("h3")
             .find_next_sibling("ul")
             .find_next_siblings()
         )
 
-        instructions = []
+        instructions: List[str] = []
         for element in elements_after_title:
             if instructions and element.name != "p":
                 break
             if element.name == "p":
-                instructions.append(element)
+                instructions.append(element.get_text())
+            instructions = [
+                normalize_string(instruction) for instruction in instructions
+            ]
 
-        return "\n".join(
-            [normalize_string(instruction.get_text()) for instruction in instructions]
-        )
+        return "\n".join(instructions) if instructions else None
```

Comment:
I'm not exactly sure how, yet, but I think/hope we'll be able to find a more Pythonic way to avoid repeated 'if x else None` statements like this.  In some cases `or None` might help, but overall I think there should be a way for us to reach `return x` (single statement without trailing conditionals) eventually.  We don't have to do that in this pull request or the near future, it's just something to think about.

Reply:
I'll admit that I'm unfamiliar with all the ins and outs of using `or` like that. I'll take a look. Some of the ternary statements are important since `0.0` and `0` are `falsey`.

```python
>>> "" or None
>>> 0 or None
>>> None or 0
0
>>> None or ""
''
>>> [] or None
>>> None or []
[]
```

Reply:
There's probably a more formal description, but it's basically: is the left-hand-side truthy?  If so, return that.  Otherwise, return the right hand side.